### PR TITLE
docs: add custom symbol demo

### PR DESCRIPTION
**Pull request by wangxingkang**
##### Checklist

- [ ] `npm test` passes
- [ ] benchmarks are included
- [ ] commit message follows commit guidelines
- [ ] documents are updated

### CODE REVIEW

**Comment by wangxingkang** [comment]
image

**Comment by pearmini** [comment]
@wangxingkang 可以把截图中的 tooltip 去掉吗！

File: site/examples/component/legend/demo/symbol.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      ['L', x - radius, y - r],
+      ['A', radius, radius, 0, 0, 0, x - r, y - radius],
+      ['L', x - r, y + radius],
+      ['A', radius, radius, 0, 0, 0, x - radius, y + r],
+      ['L', x + radius, y + r],
+      ['A', radius, radius, 0, 0, 0, x + r, y + radius],
+      ['L', x + r, y - radius],
+      ['A', radius, radius, 0, 0, 0, x + radius, y - r],
+      ['Z'],
+    ];
```
Comment (pearmini) [inline site/examples/component/legend/demo/symbol.ts:18]: 🎉

**Review by pearmini — CHANGES_REQUESTED** [review]

File: site/examples/component/legend/demo/symbol.ts (hunk 1 above)
Comment (wangxingkang) [reply site/examples/component/legend/demo/symbol.ts:18]: 好的

**Comment by wangxingkang** [comment]
image

**Comment by wangxingkang** [comment]
@pearmini 看下是否可以

**Comment by pearmini** [comment]
@wangxingkang 可以的，我周一上传一下！

**Comment by pearmini** [comment]
@wangxingkang 

https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*UM2yQKYXczEAAAAAAAAAAAAADmJ7AQ/original

**Comment by wangxingkang** [comment]
@pearmini 可以了

**Review by pearmini — APPROVED** [review]
gooood!

**Pull request merged**